recipes/systemc/2.3.3/conanfile.py

```diff
@@ -0,0 +1,114 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class SystemcConan(ConanFile):
+    name = "systemc"
+    version = "2.3.3"
+    description = """SystemC is a set of C++ classes and macros which provide
+                     an event-driven simulation interface."""
+    homepage = "https://www.accellera.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("simulation", "modeling", "esl", "tlm")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "disable_async_updates": [True, False],
+        "disable_copyright_msg": [True, False],
+        "disable_virtual_bind": [True, False],
+        "enable_assertions": [True, False],
+        "enable_immediate_self_notifications": [True, False],
+        "enable_pthreads": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+        "disable_async_updates": False,
+        "disable_copyright_msg": False,
+        "disable_virtual_bind":  False,
```

Comment:
Much more readable when double negation is avoided
```suggestion
        "enable_async_updates": True,
        "enable_copyright_msg": True,
        "enable_virtual_bind":  True,
```

Reply:
Just followed the cmake definition logic and I rather keep it inline with that else if you would look up the SystemC cmake documentation to see what the flags is doing you always need to keep in mind that it is invert in the conan options.

Reply:
Fair enough.